### ISSUE TITLE
Example of Object to RDF mappings?

### ISSUE THREAD

**Issue by wikier** [cosminbasca/surfrdf#10]
As far as I can see in the documentation, SuRF is more focused on retrieving RDF resources and map them into Python objects, right @cosminbasca?

I guess the opposite use case is also somehow supported: serialize Python object to RDF. I wonder if there is any documentation or source example covering that aspect. What do the Python classes need to provide? For instance I know they need to provide some kind of logic in the `find() methods; so I could continue with a fail-and-error approach, but i guess is not very production.

Thaks @cos

**Comment by cosminbasca** [comment]
yes, you are correct on both points, the main design purpose behind SuRF was to offer RDF access in a pythonic manner. While the reverse is possible, it is not properly documented or explored for that matter. An interesting direction to look into though (as time permits :) )

**Comment by wikier** [comment]
Can you just provide me all what you have in the project (a class example, some usage with the store, etc...), and then I'm happy to help with the documentation and possible missing features.

**Comment by cosminbasca** [comment]
sounds good, will try to find some time and crop up a few examples to get you started. Thanks again!

**Comment by charbull** [comment]
Hello, 
This library is very very interesting ! 
@cosminbasca @wikier I am interested as well in this feature as well, did you make progress on this?

Thank you :)
